compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -127,6 +127,13 @@ featureToTextContractId = Feature
     , featureCppFlag = "DAML_TO_TEXT_CONTRACT_ID"
     }
 
+featureChoiceObservers :: Feature  -- issue #7709
+featureChoiceObservers = Feature
+    { featureName = "Choice observers"
+    , featureMinVersion = versionDev
```

Comment:
Can you please leave a comment
```
- TODO Change as part of #7139
```
(like above) and add choice observers to that issue.

Reply:
done